bin/ovn-k8s-overlay

```diff
@@ -162,12 +162,16 @@ def create_management_port(node_name, local_subnet, cluster_subnet):
               "--", "set", "logical_switch", node_name,
               "other-config:subnet=" + local_subnet)
 
+    # Create route to kubernetes services
+    ovn_nbctl("--may-exist", "lr-route-add", node_name,
```

Comment:
create_management_port is called both by master_init and minion_init - in the latter case node_name will be the name of the "minion" node - and there won't be a OVN router with that name, I think